tests/Sniffs/TypeHints/ReturnTypeHintSniffTest.php

```diff
@@ -22,7 +22,7 @@ public function testErrors(): void
 			'traversableTypeHints' => ['Traversable'],
 		]);
 
-		self::assertSame(31, $report->getErrorCount());
+		self::assertSame(32, $report->getErrorCount());
```

Comment:
I missed that you did not add `assertSniffError` to check the error.

Reply:
Done, was concentrating on wrong fixer behavior.